src/support/qsort_r.c

```diff
@@ -0,0 +1,10 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
```

Comment:
Do we need the FreeBSD public domain license here?

Reply:
I don't see a reason to? It's trivial and the technique is widely used (e.g. https://git.savannah.gnu.org/cgit/coreutils.git/tree/src/false.c)

Reply:
Seems reasonable.